src/ffi/moduleobject.rs

```diff
@@ -85,6 +91,10 @@ pub struct PyModuleDef {
     pub m_free: Option<freefunc>,
 }
 
+/// Helper initial value of [`PyModuleDef`] for a Python class.
+///
+/// Not present in the Python C API.
+#[deprecated(note = "not present in Python headers; to be removed")]
 pub const PyModuleDef_INIT: PyModuleDef = PyModuleDef {
```

Comment:
This would be breaking, but I guess the only user is `pyclass.rs` and we can remove this.

Reply:
Agreed, as I think we're going to do a breaking release next anyway it's fine to just scrap this.

Reply:
@davidhewitt You'll do this in #1426 or should I add to this PR?

Reply:
I think probably simpler for you to do it in this PR so there's no risk of merge conflicts.

Reply:
Actually nvm, I see that other than that, this PR is ready to go. I'll merge this PR now and do it in #1426.